tests/test_sessions.py

```diff
@@ -120,6 +195,35 @@ def test_no_thread_on_shutdown_no_errors(sentry_init):
         environment="not-fun-env",
     )
 
+    # make it seem like the interpreter is shutting down
+    with mock.patch(
+        "threading.Thread.start",
+        side_effect=RuntimeError("can't create new thread at interpreter shutdown"),
+    ):
+        with sentry_sdk.isolation_scope() as scope:
+            with track_session(scope, session_mode="request"):
+                try:
+                    raise Exception("all is wrong")
+                except Exception:
+                    sentry_sdk.capture_exception()
+
+        with sentry_sdk.isolation_scope() as scope:
+            with track_session(scope, session_mode="request"):
+                pass
+
+        sentry_sdk.get_isolation_scope().start_session(session_mode="request")
+        sentry_sdk.get_isolation_scope().end_session()
+        sentry_sdk.flush()
```

Comment:
```suggestion
        sentry_sdk.flush()
        
        # If we reach this point without error, the test is successful.
```

Reply:
Can you also add this comment to the next test? (can not suggest this change from the github ui)

Reply:
Accidentally merged before fixed, opened #3430 with this change